lib/ide/parser/tests/ast.rs

```diff
@@ -0,0 +1,62 @@
+//! Tests specific to Ast rather than parser itself but placed here because they depend on parser
+//! to easily generate test input.
+
+use parser::prelude::*;
+
+use ast::HasRepr;
+use ast::opr;
+use ast::prefix;
+use parser::api::IsParser;
+use wasm_bindgen_test::wasm_bindgen_test;
+
+wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
+
+#[wasm_bindgen_test]
+pub fn flatten_prefix_test() {
```

Comment:
This looks wrong to me, to have tests for functions in AST crate in parser crate..